### PR TITLE
V22F-658 accel decision is not fully implemented

### PR DESCRIPTION
**Pull request by Meaganium**
Unit tests for AccelDecision implementation pass.

### CODE REVIEW

File: speed-dreams/source-2.2.3/data/data/indicators/config.xml [hunk 1]
```diff
@@ -70,5 +70,22 @@
...
+				<attstr name ="content"   val="Brake"/>
+				<attnum name ="xpos"      val="0.75"/>
+				<attnum name ="ypos"      val="0"/>
+				<attstr name ="font"      val="jura-normal.glf"/>
+				<attnum name ="font-size" val="300"/>
+			</section>
+			<section name="sound">
+				<attstr name ="source" val="warning_brake_brake.wav"/>
+			</section>
+		</section>
```
Comment (larsdekwant) [inline speed-dreams/source-2.2.3/data/data/indicators/config.xml:89]: Don't use actual tabs, replace them with spaces. (I advice you to just set this up so your IDE automatically replaces tabs by 4 spaces)

File: speed-dreams/source-2.2.3/data/data/indicators/config.xml [hunk 2]
```diff
@@ -70,5 +70,22 @@
                 <attstr name ="source" val="warning_brake_brake.wav"/>
             </section>
         </section>
+		<section name ="accelerate">
+			<section name="texture">
+				<attstr name ="source" val="Intervention_Brake_Proto_256.png"/>
```
Comment (larsdekwant) [inline speed-dreams/source-2.2.3/data/data/indicators/config.xml:75]: Maybe ask the artists whether they have a fitting texture? Adding a brake texture to the accel decision is not really a good idea.

**Review by larsdekwant — CHANGES_REQUESTED** [review]

File: speed-dreams/source-2.2.3/data/data/indicators/config.xml (hunk 2 above)
Comment (Meaganium) [reply speed-dreams/source-2.2.3/data/data/indicators/config.xml]: Hier maken we waarschijnlijk een aparte bug van in Jira

File: speed-dreams/source-2.2.3/data/data/indicators/config.xml [hunk 3]
```diff
@@ -70,5 +70,22 @@
...
             </section>
         </section>
+        <section name ="accelerate">
+            <section name="texture">
+                <attstr name ="source" val="Intervention_Brake_Proto_256.png"/>
+                <attnum name ="xpos"   val="0.75"/>
+                <attnum name ="ypos"   val="0.03"/>
+            </section>
+            <section name="text">
+                <attstr name ="content"   val="Brake"/>
```
Comment (larsdekwant) [inline speed-dreams/source-2.2.3/data/data/indicators/config.xml:80]:
```suggestion
                <attstr name ="content"   val="Accel"/>
```

**Review by larsdekwant — CHANGES_REQUESTED** [review]

File: speed-dreams/source-2.2.3/data/data/indicators/config.xml (hunk 2 above)
Comment (larsdekwant) [reply speed-dreams/source-2.2.3/data/data/indicators/config.xml]: Alright, remove the texture and sound data for now until the accel texture and sounds are available. Indicator will then only have a text displayed (for now).

**Review by larsdekwant — APPROVED** [review]

**Review by suppergerrie2 — APPROVED** [review]

**Pull request merged**